arangod/Pregel/Conductor/States/ComputingState.cpp

```diff
@@ -25,14 +28,64 @@ Computing::~Computing() {
 }
 
 auto Computing::run() -> std::optional<std::unique_ptr<State>> {
-  do {
-    conductor._preGlobalSuperStep();
+  conductor._timing.gss.emplace_back(Duration{
+      ._start = std::chrono::steady_clock::now(), ._finish = std::nullopt});
+
+  conductor._preGlobalSuperStep();
+
+  auto runGlobalSuperStepCommand = _runGlobalSuperStepCommand();
```

Comment:
Can this be made const?
```suggestion
  auto const runGlobalSuperStepCommand = _runGlobalSuperStepCommand();
```